TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+
+namespace ConsoleApp1
+{
+    class ColorPair
+    {
+        public Color majorColor;
+        public Color minorColor;
+        public override string ToString()
+        {
+            return string.Format("MajorColor:{0}, MinorColor:{1}", majorColor.Name, minorColor.Name);
+        }
+        public static int GetPairNumberFromColor(ColorPair pair)
```

Comment:
Why is `GetPairNumberFromColor` inside `ColorPair`, but not the function to get Color-pair from number?

Reply:
Each class performs only one function.Hence colorPair class retrieves only the pairnumber from the the given color and not vice versa.